.github/workflows/global-ci-bundle.yml

```diff
@@ -14,6 +14,13 @@ on:
         type: string
         required: false
         default: ""
+      operator_branch:
```

Comment:
Not sure if this needs to be adjusted also in `bundle`, but added as optional/with default value should not  break it.